js/bars.js

```diff
@@ -1259,13 +1301,13 @@ historian.Bars.prototype.tooltipText_ = function(
     }
   } else {
     // Attach a div that the table can be appended to.
-    // The table shouldn't be appended here as we may just want the text
-    // output for copying the tooltip.
+    // The table shouldn't be appended here as only the text output for 
```

Comment:
One last one on line 1304

Reply:
Can you please check the lastest commit with comment "space", the one with ID 276c080?